github/util.py

```diff
@@ -235,9 +251,13 @@ def _create_github_api_object(
     if not github_api:
         util.fail("Could not connect to GitHub-instance {url}".format(url=github_url))
 
```

Comment:
add a comment, maybe?
e.g. # patch github's requests.session to retry upon sporadic errors